client/views/curatorInbox/sourceIncidentReportsHeader.jade

```diff
@@ -0,0 +1,22 @@
+template(name="sourceIncidentReportsHeader")
+  .curator-source-details--sub-header
+    button.btn.btn-sm.btn-danger.delete(
+      disabled="{{#unless incidentsSelectedCount}} true {{/unless}}")
+      if deleting
```

Comment:
I don't think we need a deleting loader. We can use latency compensation to prevent delays.